cargodocs/centralized.md

```diff
@@ -0,0 +1,110 @@
+Telescope construction with prehashing and bounded DFS scheme given in _Section 3.2.2_.
+
+
+In some cases, $n_p$ may be small, which can affect the completeness and efficiency of the proof construction. The protocol adapts to such cases by introducing optimizations and tighter bounds.
+- In the basic ALBA scheme, optimal performance requires $n_p \geq \lambda^3$.
+- When $n_p$ is small (e.g., $n_p < \lambda^3$), the probability of successfully finding valid proof paths diminishes.
+- Ensure that the proof remains efficient and sound even when $n_p$ is small by introducing an improved protocol with tighter guarantees on the number of attempts and running time.
+- When $n_p$ is too small to guarantee a valid proof in a single attempt, the protocol allows the prover to make multiple attempts.
+- Each attempt is indexed by an integer $v \in \{1, \ldots, r\}$, where $r$ is the number of repetitions. This ensures higher completeness by amplifying the success probability.
+  - Set $r := \lambda_{\text{rel}}$, ensuring the completeness error becomes $2^{-\lambda_{\text{rel}}}$.
+  - This approach maintains completeness even when the prover has a limited number of elements in their set.
+
+### Optimized Proof Construction
+- A valid proof now consists of a tuple $(v, t, s_1, \ldots, s_u)$, where:
+  - $v$ is the repetition index.
+  - $t \in \{1, \ldots, d\}$ is another index used to explore proof paths.
```

Comment:
we should say what `d` is - also, what is a good way of explaining what `exploring proof paths` mean?